workos/resources/base.py

```diff
@@ -19,7 +19,7 @@ def construct_from_response(cls, response):
         """
         obj = cls()
         for field in cls.OBJECT_FIELDS:
-            setattr(obj, field, response[field])
+            setattr(obj, field, response[field] if field in response else None)
```

Comment:
`response[field] if field in response else None` can be simplified to `response.get(field)` which will default to `None`.